src/utils/pagerduty/resolve-incidents.ts

```diff
@@ -0,0 +1,52 @@
+/* eslint-disable no-console */
+
+import {sleep} from '../sleep'
+
+export const resolveIncidents = async (
+  pdClient: any,
+  serviceId: string,
+  testService: string,
+) => {
+  const allIncidents = await pdClient.all(
+    `/incidents?service_ids%5B%5D=${serviceId}&statuses%5B%5D=acknowledged&statuses%5B%5D=triggered`,
+  )
+  const allOpenIncidents = allIncidents.data
+  .reduce((acc: any, i: any) => {
+    if (i.incidents !== undefined && i.incidents.length > 0) {
+      acc = [...acc, ...i.incidents]
+    }
+    return acc
+  }, [])
+  .filter((i: any) => i.title.split(' - Tests: ')[0] === testService)
+  console.log(
+    `${allOpenIncidents.length} incidents still open in pagerduty for service: ${testService}`,
+  )
+  if (allOpenIncidents.length > 0) {
+    console.log('These incidents will be resolved')
+  }
+  for (const incident of allOpenIncidents) {
+    // eslint-disable-next-line no-await-in-loop
+    const incidentResponse = await pdClient.put(`/incidents/${incident.id}`, {
+      data: {
+        incident: {
+          type: 'incident',
+          status: 'resolved',
+          resolution:
+            'Incident was automatically resolved by jahia-reporter since the last run was successful',
```

Comment:
Is it possible to add a link or an ID of the success run that trigger that incident to be closed?